packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -34,6 +34,9 @@ import 'variables.dart';
 final debuggerCodeViewSearchKey =
     GlobalKey(debugLabel: 'DebuggerCodeViewSearchKey');
 
+// This is a conservative estimate, most fonts have a width/height ratio of ~0.5.
+const approximateFontWidthHeightRatio = 0.6;
```

Comment:
Rather than guessing, you can draw a character (e.g. ' ') and measure what the ratio is.
as the console is assuming a fixed width font, that is safe.
Existing example:
https://github.com/flutter/devtools/blob/b9950aa0c6af0e095f2c5b254f41a127c9481a70/packages/devtools_app/lib/src/common_widgets.dart#L1149

Reply:
Addressing this comment will help the content be the correct size.